src/integrant/core.cljc

```diff
@@ -13,10 +13,20 @@
 (defrecord Ref    [key] RefLike (ref-key [_] key))
 (defrecord RefSet [key] RefLike (ref-key [_] key))
 
+(def ^{:private true} composite-key? vector?)
```

Comment:
I don't think we need this. If anything, it requires an extra lookup whenever anyone sees `composite-key?`, making it less clear to read.